sidebars.js

```diff
@@ -276,11 +276,18 @@ module.exports = {
       label: 'Integration guides',
       link: { type: 'doc', id: 'deployment_patterns/integrations_and_howtos_overview' },
       items: [
+        {
+          type: 'category',
+          label: 'Using Great Expectations with AWS',
+          items: [
+            'deployment_patterns/how_to_use_great_expectations_in_aws_glue',
+            { type: 'doc', id: 'deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_cloud_storage_and_pandas' }
```

Comment:
this renders as "Amazon Web Services" in the TOC, which is needlessly long. I think the nesting is fine, I could see us expanding this section substantially over time (as a good thing!)